modules/i18n/src/translations/en-US/portals/console.ts

```diff
@@ -6391,32 +6391,97 @@ export const console: ConsoleNS = {
                                 }
                             },
                             verifyHostname: {
-                                label: "Verify the hostname"
+                                label: "Verify the hostname",
+                                hint: "Enable this option to verify the hostname of the remote server."
```

Comment:
Would it be possible to enhance the hint by explaining what happens when this option is enabled, rather than simply repeating the label text?